examples/sumo/minicity.py

```diff
@@ -96,10 +96,10 @@ def minicity_example(render=None,
     # Dynamic grayscale rendering: minicity_example(render="dgray")
     # Static RGB rendering: minicity_example(render="rgb")
     # Dynamic RGB rendering: minicity_example(render="drgb")
-    exp = minicity_example(render=True,
-                           save_render=False,
-                           sight_radius=50,
-                           pxpm=3,
+    exp = minicity_example(render='drgb',
+                           save_render=True,
```

Comment:
the example should not auto-save the render. This is sort of unpleasant unexpected behavior.